lib/gooddata/models/membership.rb

```diff
@@ -14,7 +14,7 @@
 require_relative '../rest/object'
 
 module GoodData
-  class Membership < Rest::Resource
+class Membership < Rest::Resource
```

Comment:
warning: Layout/IndentationWidth: Use 2 (not 0) spaces for indentation. (rubocop)